controllers/freebasics/forms.py

```diff
@@ -0,0 +1,53 @@
+from controllers.docker.forms import DockerControllerForm
+from controllers.freebasics.models import FreeBasicsController
+from django import forms
+from django.forms.utils import ErrorList
+
+TEMPLATE_CHOICES = (
```

Comment:
You can move these to `line 20` so that they can be referenced using `FeeBasicsControllerForm.TEMPLATE_CHOICES`